index.ts

```diff
@@ -287,22 +287,28 @@ collect.set('isRepoIssueList', [
 	'https://github.com/sindresorhus/refined-github/issues?q=is%3Aclosed+sort%3Aupdated-desc',
 ]);
 
-export const isRepoRoot = (url: URL | Location = location): boolean => /^(tree\/[^/]+)?$/.test(getRepoPath(url)!);
-collect.set('isRepoRoot', [
+export const isRepoHome = (url: URL | Location = location): boolean => isRepoRoot(url) && getRepoPath(url) === '';
```

Comment:
```suggestion
export const isRepoHome = (url: URL | Location = location): boolean => getRepoPath(url) === '';
```

Dont thing that is needed